src/main/java/org/jboss/snowdrop/licenses/sanitiser/ExceptionLicenseSanitiser.java

```diff
@@ -38,11 +39,9 @@ public ExceptionLicenseSanitiser(String exceptionsFilePath, LicenseSanitiser nex
     @Override
     public DependencyElement fix(DependencyElement dependencyElement) {
         return exceptions.parallelStream()
-                .filter(dependencyElement::equals)
```

Comment:
I explicitly ignored all the suffixes to make service more reusable and not require to hardcode `redhat` strings. Adding two exceptions to the exceptions file solves the issue. Is this a limitation for the EAP licenses service?

Reply:
yes, eap service currently only serves data for -redhat- artifacts